test/parallel/test-buffer-isencoding.js

```diff
@@ -17,7 +17,8 @@ const assert = require('assert');
     assert.strictEqual(Buffer.isEncoding(enc), true);
   });
 
-[ 'utf9',
+[ '',
+  'utf9',
```

Comment:
I have no idea, I just did a quick check, and `''` was accepted as an encoding, but you said my node version was not master, so perhaps my check was wrong? I'm not entirely clear on what the purpose of the API is, but if it is intended to find encoding values that don't work, it should agree with usage.

So, are various Buffer APIs inconsistent? If so, that would be a good thing to fix! :-)